ansible_collections/arista/avd/roles/eos_cli_config_gen/tasks/build-documentation.yml

```diff
@@ -24,6 +24,7 @@
   template:
     src: avd_schema_documentation.j2
     dest: "{{ role_documentation_dir ~ '/' ~ documentation_file.filename ~ '.md'  }}"
+    mode: 0775
```

Comment:
I don't think these are correct. We should not have execute on data files.
We need 664.

Reply:
fixed in 032e20e